### DISCUSSION TITLE
Add "Input element with time picker" to ui.time documentation

### DISCUSSION THREAD

**Discussion by leonstafford** [zauberzeug/nicegui#2675] [Ideas / Feature Requests]
I'll be attempting this soon for my own needs and happy to submit a PR if I can produce an elegant example.

@falkoschindler, any tips or basically same as the `ui.date()` example?

**Comment by leonstafford** [comment]
This seems to work fine for me:

```python
with ui.input(label='Time', value='14:00') as dropoff_time:
    with dropoff_time.add_slot('append'):
        ui.icon('schedule').on('click', lambda: menu()).classes('cursor-pointer')
    with ui.menu() as menu:
        ui.time().bind_value(dropoff_time)
```

**Reply by falkoschindler** [reply]
Looks good, @leonstafford! Just a few thoughts:

1. You should call `menu.open()` instead of `menu()`.
2. We can eliminate the lambda expression by moving it up and simply writing `.on('click', menu.open)`.
3. We should add `.props('no-parent-event')` so that the menu doesn't automatically open when clicking into the input field.

```py
with ui.input(label='Time', value='14:00') as time:
    with ui.menu().props('no-parent-event') as menu:
        ui.time().bind_value(time)
    with time.add_slot('append'):
        ui.icon('schedule').on('click', menu.open).classes('cursor-pointer')
```

I think we should update the `ui.date` demo accordingly. Would you like to do that in your PR? 🙂

**Reply by leonstafford** [reply]
Thanks, @falkoschindler ! Yes, I'll send a  PR addressing those soon.
